kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -69,9 +74,19 @@ class PARTICIPANT(PropertyMixin):
         SPECIES = None
         ENROLLMENT_AGE_DAYS = None
 
+        class ENROLLMENT_AGE(QuantityMixin):
+            pass
+
+        class LAST_CONTACT(QuantityMixin):
```

Comment:
Why is this a quantity? Is it supposed to be an age as well?

Reply:
I think everything is supposed to be an age in KF?

Reply:
Since all of our other age fields have the AGE suffix, should we add it to this one too?